client/lib/cart-values/cart-items.js

```diff
@@ -395,6 +399,13 @@ function domainRegistration( properties ) {
 	return assign( domainItem( properties.productSlug, properties.domain ), { is_domain_registration: true } );
 }
 
+domainRegistration.fromSuggestion = function( suggestion ) {
```

Comment:
I remember there were a lot of calls where we had to convert props from underscore to camelCase  and I was sick of it.
Removed for now.